cmake/Summary.cmake

```diff
@@ -125,7 +120,10 @@ function (caffe2_print_configuration_summary)
   message(STATUS "  USE_ZMQ               : ${USE_ZMQ}")
   message(STATUS "  USE_DISTRIBUTED       : ${USE_DISTRIBUTED}")
   if(${USE_DISTRIBUTED})
-    message(STATUS "    USE_DISTRIBUTED_MW     : ${USE_DISTRIBUTED_MW}")
+    message(STATUS "    USE_DISTRIBUTED_MW  : ${USE_DISTRIBUTED_MW}")
```

Comment:
I think `DISTRIBUTED_MW` can go. This has been disabled for a while and I doubt it still compiles. If it hasn't been removed from the repo altogether yet.

Reply:
Should this be removed as well? https://github.com/pytorch/pytorch/blob/master/torch/csrc/Module.cpp#L602-L612

Reply:
Yes, good catch! See #10731 for corresponding PR that removed the master worker stuff.

Reply:
@orionr See #11240